README.md

```diff
@@ -106,6 +107,11 @@ module "my_rds_instance" {
     subnets = ["${aws_subnet.example.*.id}"] # see above
     rds_vpc_id = "${module.vpc}"
     private_cidr = "${var.private_cidr}"
+
+    tags {
+        "Terraform" = "true"
```

Comment:
Quotes around tag names are not necessarry.
Also, just a tought: It would be extremely useful to create a map of tags as a variable and expand it to tags in this place.Eg. instead of "Env" I'm using "environment" and also have couple of more tags.
I don't have any example at the moment, but will try to work something out in a spare time.